src/bot.ts

```diff
@@ -133,37 +152,42 @@ dfusionService.watchOrderPlacement({
-    const sellTokenLabel = sellToken.symbol || sellToken.name || sellToken.address
-    const buyTokenLabel = buyToken.symbol || buyToken.name || buyToken.address
-
-    // Only display the valid from if the period hasn't started
-    const now = new Date()
-    let datesDescription = ''
-    if (validFrom > now) {
-      // The order is not active yet
-      datesDescription = `  - *Tradable*: \`${moment(validFrom).calendar()} GMT\`, \`${moment(validFrom).fromNow()}\`\n`
-    }
-    datesDescription += `  - *Expires*: \`${moment(validUntil).calendar()} GMT\`, \`${moment(validUntil).fromNow()}\``
+    const { tokenLabel: sellTokenLabel, tokenParam: sellTokenParam, amountFmt: sellAmountFmt } = _getTokenFmt(
+      priceDenominator,
+      sellToken
+    )
+    const { tokenLabel: buyTokenLabel, tokenParam: buyTokenParam, amountFmt: buyAmountFmt } = _getTokenFmt(
+      priceNumerator,
+      buyToken
+    )
 
     // Format the amounts
     // TODO: Allow to use BN, string or BigNumber or all three in the format. Review in dex-js
-    const sellAmountFmt = formatAmount(new BN(priceDenominator.toString()), sellToken.decimals)
-    const buyAmountFmt = formatAmount(new BN(priceNumerator.toString()), buyToken.decimals)
     const fillSellAmountFmt = formatAmountFull(
       new BN(priceNumerator.multipliedBy(FACTOR_TO_FILL_ORDER).toString()),
       buyToken.decimals
     )
     const buyAmountFullFmt = formatAmountFull(new BN(priceDenominator.toString()), sellToken.decimals)
 
-    // Compose message using markdown
-    // TODO: Provide the link to the front end: https://github.com/gnosis/dex-telegram/issues/3
     // TODO: Should we publish even if the user doesn't have balance. Should we include the balance of the user? he can change it...
-    const message = `Sell *${sellAmountFmt}* \`${sellTokenLabel}\` for *${buyAmountFmt}* \`${buyTokenLabel}\`
-
-  - *Price*:  1 \`${sellTokenLabel}\` = ${price} \`${buyTokenLabel}\`
-${datesDescription}
+    //  https://github.com/gnosis/dex-telegram/issues/45
+    let message = `Sell *${sellAmountFmt}* \`${sellTokenLabel}\` for *${buyAmountFmt}* \`${buyTokenLabel}\`\n`
+    message += `\n  - *Price*:  1 \`${sellTokenLabel}\` = ${price} \`${buyTokenLabel}\``
 
-Fill the order here: ${WEB_BASE_URL}/trade/${buyTokenLabel}-${sellTokenLabel}?sell=${fillSellAmountFmt}&buy=${buyAmountFullFmt}` // TODO:
+    // Only display the valid from if the period hasn't started
+    const now = new Date()
+    if (validFrom > now) {
+      // The order is not active yet
+      message += `\n  - *Tradable*: \`${moment(validFrom).calendar()} GMT\`, \`${moment(validFrom).fromNow()}\``
+    }
+    message += `\n  - *Expires*: \`${moment(validUntil).calendar()} GMT\`, \`${moment(validUntil).fromNow()}\``
+    if (!sellToken.known || !buyToken.known) {
+      message +=
+        '\n  - "Maybe <token-symbol>" means the token is not recognizable as a token in the following [web](' +
```

Comment:
This sentence is a bit weird.

Suggesting a bit longer version:

"Maybe <token symbol>" means one or more tokens claim to be <token symbol>, but it's not currently part of our list of known tokens. Make sure you verify the address yourself before trading against it.

Reply:
Good input. I'll change `our` for the `bot's` to don't show belonging of any kind